src/GeneralStateTestsFiller/stCreateTest/CREATE_EOF1InvalidOpcodesFiller.yml

```
@@ -0,0 +1,176 @@
+# Failes to deploy EOF code containing each one of Invalid Opcodes
+CREATE_EOF1InvalidOpcodes:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1000000000
+      code: ''
+      nonce: 0
+      storage: {}
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      code: |
+        :yul
+        {
+          // store opcode from call data into memory
+          mstore(0, calldataload(0))
+
+          // initcode: { mstore(0, 0xef0001010002000000) return(23, 9) } }
+          mstore(128, 0x68ef000101000200000060005260096017f30000000000000000000000000000)
+
+          // Add the opcode to the init code
+          let code := shr(248, mload(0))
```

couldn't you just calldataload here directly?
```suggestion
          let code := shr(248, calldataload(0))
```